src/test/java/com/fundation/webservice/model/CriteriaKeyFrameVideoTest.java

```diff
@@ -0,0 +1,33 @@
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CriteriaKeyFrameVideoTest {
+
+    CriteriaKeyFrameVideo criteriaTest = new CriteriaKeyFrameVideo();
+    @Before
+    public void setUp(){
+        criteriaTest.setSrcPath(Directories.RSRC_DIR.getDir()+"videoTest.ogg");
+        criteriaTest.setDestPath(Directories.RSRC_DIR.getDir());
+        criteriaTest.setFrames("1000");
+        criteriaTest.setName("output");
+        criteriaTest.setExt("mp4");
+    }
+    @Test
```

Comment:
Please insert an additional line to space between both methods.

Reply:
Okay, it's fixed.